source/get-started/install-configfile.rst

```diff
@@ -131,54 +125,51 @@ Perform the following steps:
 
 .. _install-configfile-yaml-end:
 
-Start the installation with the command:
+#. Start the unattended installation using the `--config` option.
 
-.. code-block:: bash
+   .. code-block:: bash
 
-   clr-installer --config live-server.yaml
+      clr-installer --config live-server.yaml
 
-Example 2
-=========
+Example 2: Replicate a previous installation
+********************************************
 
-This method uses a saved configuration file from a previous installation,
+This example uses a saved configuration file from a previous installation,

```

Comment:
Please make a direct link to this document: 
"... previous installation, as shown in : https://docs.01.org/clearlinux/latest/get-started/bare-metal-install-server.html#save-configuration-settings

Reply:
it's not necessary to link this document.  clr-installer saves the YAML of an installed system to /root regardless.